src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -176,20 +176,29 @@ export const BankInfoCNP = ({
 
   // When direct deposit is not set up, we will show
   const notSetUpContent = (
-    <button
-      className="va-button-link"
-      ref={editBankInfoButton}
-      onClick={() => {
-        recordEvent({
-          event: 'profile-navigation',
-          'profile-action': 'add-link',
-          'profile-section': 'cnp-direct-deposit-information',
-        });
-        toggleEditState();
-      }}
-    >
-      Please add your bank information
-    </button>
+    <div className={classes.bankInfo}>
+      <dl className="vads-u-margin-y--0 vads-u-line-height--6">
```

Comment:
I don't think that this needs to be a `dl` when it's the "empty" state. A `span` should be fine, which is what we use when a piece of contact info is not set. https://github.com/department-of-veterans-affairs/vets-website/blob/0d94a4851950d9a01bd4a5482c3f02509041891b/src/applications/personalization/profile/components/personal-information/ContactInformationView.jsx#L19-L21

Reply:
I was mainly trying to keep the same structure as the other places here, but I can change it to that if it looks the same and it's what we need here